cves/CVE-2012-2774.yml

```diff
@@ -257,8 +272,8 @@ interesting_commits:
       * A change that looks really sketchy
       * Commits where a developer mentions security in their message or comments
   commits:
-  - commit:
-    note:
+  - commit: 61c2cec9579b5a24e95102b137eab3ecc6902d15
+    note: This one has a spelling mistake in the commit message.
```

Comment:
I'm not sure that makes it interesting per se. If you didn't find an interesting commit I think the tests will let you pass anyway.